projects/client/RabbitMQ.Client/src/client/content/BytesMessageReader.cs

```diff
@@ -43,12 +43,12 @@ namespace RabbitMQ.Client.Content
     /// <summary>
     /// Analyzes AMQP Basic-class messages binary-compatible with QPid's "BytesMessage" wire encoding.
     /// </summary>
-    public class BytesMessageReader : BasicMessageReader, IBytesMessageReader
+    public class BytesMessageReader : BasicMessageReader//, IBytesMessageReader
```

Comment:
Why is the interface commented out? Is it no longer necessary to implement the interface?

In that case it should be deleted rather than commented out.